.docker/Dockerfile

```diff
@@ -40,7 +42,7 @@ FROM ci as robot
 
 # Configure a new non-root user
 ARG USERNAME=blue
-ARG USER_UID=1000
+ARG USER_UID=1001   # ros image now includes a user "ubuntu" at UID 1000
```

Comment:
You may run into permissions issues here when trying to use the devcontainer. The blue user likely won't belong to the same group as your personal user on your local system (set to 1000 by default). My solution has been to go along with the ubuntu user standardization.

Reply:
Hm, I hadn't considered the permission ramifications.   So you are using the user `ubuntu`?

```
# Configure a new non-root user
#
# ros image now includes a user "ubuntu" at UID 1000
ARG USERNAME=ubuntu
ARG USER_UID=1000  
ARG USER_GID=$USER_UID

RUN echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME \
    && chmod 0440 /etc/sudoers.d/$USERNAME \
    && usermod -a -G dialout $USERNAME \
    && echo "source /usr/share/bash-completion/completions/git" >> /home/$USERNAME/.bashrc
```

Reply:
Yeah. This was motivated by discussions in [cloud-images!2005129](https://bugs.launchpad.net/cloud-images/+bug/2005129), which make it seem like the Ubuntu devs are attempting to standardize the `ubuntu` user. You could still delete the `ubuntu` user to regain access to UID 1000, but I opted against that.